Tests/SwiftParserTest/AvailabilityTests.swift

```diff
@@ -151,7 +151,11 @@ final class AvailabilityTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(message: "expected version tuple in version restriction", fixIts: ["insert version tuple"]),
         DiagnosticSpec(message: "unexpected code '10e10' in attribute"),
-      ]
+      ],
+      fixedSource: """
+        @available(OSX <#integer literal#>10e10)
```

Comment:
We should emit a diagnostic here that `10e10` is not a valid version tuple instead of inserting an integer literal and leaving `10e10`.